File: samples/legacy-architecture-samples/simple-stack-example-mvp-view/src/main/java/com/zhuinden/simplestackdemoexamplemvp/data/entity/DbTask.java
package com.zhuinden.simplestackdemoexamplemvp.data.entity;

import io.realm.RealmObject;
import io.realm.annotations.Index;
import io.realm.annotations.PrimaryKey;

public class DbTask
        extends RealmObject {
    @PrimaryKey
    private String id;

    private String title;

    private String description;

    @Index
    private boolean completed;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public boolean getCompleted() {
        return completed;
    }

    public void setCompleted(boolean completed) {
        this.completed = completed;
    }
}

File: examples/simple-stack-example-mvp-view/src/main/java/com/zhuinden/simplestackdemoexamplemvp/data/entity/DbTask.kt
package com.zhuinden.simplestackdemoexamplemvp.data.entity

import io.realm.RealmObject
import io.realm.annotations.Index
import io.realm.annotations.PrimaryKey

/**
 * Created by Owner on 2017. 01. 26..
 */

open class DbTask : RealmObject() {
    @PrimaryKey
    var id: String? = null

    var title: String? = null

    var description: String? = null

    @Index
    var completed: Boolean = false
}
